index.js

```diff
@@ -15,164 +23,341 @@ const supportedHelpers = [
   'TestCafe',
   'Playwright'
 ];
-let launchObj;
-let itemObj;
-let fileName;
-let stepInfo;
-let rpClient;
-let logFile;
-let suiteTempId;
-let beforeSuiteStatus = 'failed';
-let launchStatus = 'passed';
-
-class ReportPortalHelper extends Helper {
-  _updateStep(step, status) {
-    this._finishTestItem(launchObj, itemObj, step, status);
-  }
 
-  _passed() {
-    this._updateStep(stepInfo, 'passed');
+for (const helperName of supportedHelpers) {
+  if (Object.keys(helpers).indexOf(helperName) > -1) {
+    helper = helpers[helperName];
   }
+}
 
-  async _failed(test) {
-    launchStatus = 'failed';
-    this.errMsg = test.err.message;
-    this._updateStep(stepInfo, 'failed');
-  }
+const defaultConfig = {
+  token: '',
+  endpoint: '',
+  project: '',
+  launchDescription: '',
+  attributes: [],
+  debug: false,
+  rerun: undefined,
+  enabled: false
+};
 
-  _startLaunch(suiteTitle) {
-    rpClient = new RPClient({
-      token: this.config.token,
-      endpoint: this.config.endpoint,
-      project: this.config.projectName,
-      debug: this.config.debug,
-    });
+module.exports = (config) => {
+  config = Object.assign(defaultConfig, config);
 
-    return rpClient.startLaunch({
-      name: this.config.launchName || suiteTitle,
-      description: this.config.launchDescription || '',
-      attributes: this.config.launchAttributes || [],
-      rerun: this.config.rerun || false,
-      rerunOf: this.config.rerunOf || undefined,
+  console.log('Staaarting!!!11');
```

Comment:
why don't we use `output` of codeceptjs here? :)

Reply:
This is a very dump debug :)